src/commands/src/helpers/dependency-helpers.ts

```diff
@@ -22,14 +15,14 @@ async function createEnvProdFile(repoDir: string, projectName: string, baseUrl:
       prodFileName,
       `GATSBY_SITE=${projectName}
       GATSBY_MANIFEST_PATH=${repoDir}/bundle.zip
-      GATSBY_PARSER_USER=${process.env.USER}
+      GATSBY_PARSER_USER=${process.env.USER ?? 'docsworker-xlarge'}
```

Comment:
This is something that I forgot to do when writing this, but we will need to conditionally include the properties `PATCH_ID` and `COMMIT_HASH` if `patchId` exists so that it matches what the [makefiles are doing](https://github.com/mongodb/docs-worker-pool/blob/12a321a5e8163074c463a79502ff07b930e7f8e9/makefiles/shared.mk#L78). 

We can provide these values after we get those dependencies, but we will have to move [the function call](https://github.com/mongodb/docs-worker-pool/blob/ffe258c96da8516743c5844fcadd209da194715e/src/commands/src/helpers/dependency-helpers.ts#L87) outside of the array so that we have `patchId` and `commitHash` as well.

For context, the `nextGenHtml` function depends on these values.